### ISSUE TITLE
Batch PUTs?

### ISSUE THREAD

**Issue by acannon828** [ticketea/pynesis#15]
The Kinesis API supports batching PutRecord requests, does/will pynesis?

**Comment by msurdi** [comment]
Yes, we'd definitely like to support this. I'll try to find some time this weekend to give it a try. Would you be able to help by trying it out (beta test) this before we merge it in?
